apps/block_scout_web/lib/block_scout_web/views/currency_helpers.ex

```diff
@@ -69,7 +69,7 @@ defmodule BlockScoutWeb.CurrencyHelpers do
 
   @spec format_according_to_decimals(Decimal.t(), Decimal.t()) :: String.t()
   def format_according_to_decimals(value, decimals) do
-    if Decimal.cmp(decimals, 18) == :gt do
+    if Decimal.cmp(decimals, 24) == :gt do
```

Comment:
not sure what is happening here, but the format should probably have the same decimal precision

Reply:
It's coming from [this issue](https://github.com/blockscout/blockscout/pull/5008). Here is the [PR](https://github.com/blockscout/blockscout/commit/b38cb83ee60212480811cac2454210089d5f5db0).